website/templates/emails/pending_retraction_non_admin.txt.mako

```diff
@@ -0,0 +1,7 @@
+Hello ${user.fullname},
+
+I just wanted to let you know, ${initiated_by} has requested a retraction for the following registration: ${registration_link}.
+
```

Comment:
My answers are:

Yes, user initiating the Retraction or Embargo should also receive the
email.

Yes, initiating the Retraction or Embargo implies approval, but message
then gives them the option of changing their minds by disapproving.
Functionally, if it makes a difference for ease of implementation, it is
fine to have the user not marked as approved initially so that they can
select approve or disapprove.  But, if a trivial difference, then
defaulting to approved is sensible.



On Sat, May 23, 2015 at 11:19 AM, Harry Rybacki <notifications@github.com>
wrote:

> In website/templates/emails/pending_retraction_non_admin.txt.mako
> <https://github.com/CenterForOpenScience/osf.io/pull/2647#discussion_r30945660>
> :
>
> > @@ -0,0 +1,7 @@
> > +Hello ${user.fullname},
> > +
> > +I just wanted to let you know, ${initiated_by} has requested a retraction for the following registration: ${registration_link}.
> > +
>
> To your aside: I followed a similar thought process. Either way we should
> agree on the behavior before I run and spend a few hours changing this
> behavior.
>
> To reiterate the idea is: Should the user initiating a Retraction or
> Embargo also receive an email? Furthermore, does the action of initiating
> the Retraction or Embargo imply their approval?
>
> @sloria <https://github.com/sloria> @JeffSpies
> <https://github.com/JeffSpies> thoughts?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CenterForOpenScience/osf.io/pull/2647/files#r30945660>
> .
>